FastSimulation/Muons/plugins/FastTSGFromL2Muon.cc

```diff
@@ -49,8 +49,9 @@ FastTSGFromL2Muon::beginRun(edm::Run const& run, edm::EventSetup const& es)
   edm::ParameterSet regionBuilderPSet = 
     theConfig.getParameter<edm::ParameterSet>("MuonTrackingRegionBuilder");
   edm::ConsumesCollector iC  = consumesCollector();
-  theRegionBuilder = new MuonTrackingRegionBuilder(regionBuilderPSet,theService,iC);
```

Comment:
Hi Lukas,

We changed the MuonTrackingRegionBuilder, so now it no longer uses the MuonServiceProxy. Although there are still calls to updating the theService in FastTSGFromL2Muon::beginRun and FastTSGFromL2Muon::produce, I guess they are not used by anything else? In which case, yes theService could be removed.

Thanks,
Benjamin